### PR TITLE
feat(domain-dkim): fix preview of dkim

### PR DESCRIPTION
**Pull request by varun257**
init values of checkbox
assign model value to checkbox

DTRSD-9480

Signed-off-by: Varun257 <varun257@gmail.com>

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
                 id="v"
                 name="v"
                 data-model="ctrl.model.target.v.DKIM1"
-                data-on-change="ctrl.setTargetValue('DKIM')"
+                data-on-change="ctrl.model.target.v.DKIM1 = modelValue; ctrl.setTargetValue('DKIM')"
```
Comment (FredericEspiau) [inline packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html]: Maybe you could extract to a function 🙂

Comment (radireddy) [inline packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html]: can you add this in controller and call that method in all other places plz, also add 'DKIM' in contants

File: packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html [hunk 2]
```diff
@@ -53,15 +53,15 @@
                 name="h"
                 id="h-sha1"
                 data-model="ctrl.model.target.h.sha1"
-                data-on-change="ctrl.setTargetValue('DKIM')"
+                data-on-change="ctrl.model.target.h.sha1 = modelValue; ctrl.setTargetValue('DKIM')"
```
Comment (radireddy) [inline packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html]: call the reusable method in controller

File: packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html [hunk 3]
```diff
@@ -53,15 +53,15 @@
...
                 data-text="SHA-1"
             >
             </oui-checkbox>
             <oui-checkbox
                 name="h"
                 id="h-sha256"
                 data-model="ctrl.model.target.h.sha256"
-                data-on-change="ctrl.setTargetValue('DKIM')"
+                data-on-change="ctrl.model.target.h.sha256 = modelValue; ctrl.setTargetValue('DKIM')"
```
Comment (radireddy) [inline packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html]: call the reusable method in controller and same in below 2 places

File: packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html (hunk 1 above)
Comment (varun257) [reply packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html]: There are a lot of constants and the whole code needs to be rewritten in future so we should do it when we refactor :)

Comment (FredericEspiau) [reply packages/manager/apps/web/client/app/domain/zone/record/DKIM/domain-zone-record-DKIM.html]: Ok @varun257 but please create a JIRA for this and add it to `TO_GROOM` sprint, this way we don't forget about this 🙂

**Pull request merged**